src/nasir/Test1_Program1.java

```diff
@@ -0,0 +1,23 @@
+package nasir;
+
+public class Test1_Program1 {
+	
+	void setString(String input) {
+		for(int index=0;index<input.length();index++) {
+			int count=0;
+			for(int index1=0;index1<input.length();index1++) {
+				if(input.charAt(index1)==input.charAt(index))
+					count++;
+			}
+			if(count==1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop